types/adeunis/pulse4/default.schema.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$id": "https://akenza.io/adeunis/pulse4/default.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Default",
+  "type": "object",
+  "properties": {
+    "counterValueT0": {
+      "title": "Counter value time - 0",
```

Comment:
what does time 0 mean? can you find a better title?

Reply:
Most recent measurement.
I renamed it to counterValue. The other keys still have the time indication.

Reply:
I mean, how is t measured?

Reply:
Well its a variable timeframe when the measurements gets taken.  So f.e if you have 10 measurements in one hour with one datapush. counterValueT1 would be (now - 6m), counterValueT2  (now - 12m) etc. If you take 6 measurements in one hour with one datapush, you would have counterValueT1  (now - 10m), counterValueT2  (now - 20m) etc